delivery_carrier_label_postlogistics/wizards/postlogistics_oauth.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+<!--
+    Copyright 2020 Camptocamp SA
+-->
+
+  <record id="view_postlogistics_auth_form" model="ir.ui.view">
+    <field name="name">postlogistics.auth.form</field>
+    <field name="model">postlogistics.auth</field>
+    <field name="arch" type="xml">
+      <form string="Postlogistics Auth">
+        <field invisible="1" name="state"/>
+
+        <!--    To do   -->
+        <group states="todo">
+          <h2>Set the swisspost client information</h2>
+        </group>
+
+        <group states="todo">
+          <field name="client_id" required="1"/>
+          <field name="client_secret" required="1"/>
+        </group>
+
+        <!--    Done    -->
+        <group states="done">
+          <h2>OAuth configuration complete</h2>
+        </group>
+
+        <footer states="todo">
+          <button name="generate_access_token" type="object" string="Generate access token"/>
+          <button special="cancel" string="Cancel"/>
+        </footer>
```

Comment:
it's todo so I thing that ca be in https://github.com/sebalix/delivery-carrier/pull/1/files#diff-19dff0e5cdd5675da5e2369ea1665803R23